bin/grax.sh

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env bash
+
+./graxctl migrate 
+./graxctl secrets -force "{\"Provider\":\"aws\",\"Type\":\"storage\",\"Name\":\"StorageSecret\",\"Data\":{\"access_key_id\":\"$S3_AWS_ACCESS_KEY_ID\",\"provider\":\"aws\",\"region\":\"$S3_AWS_REGION\",\"s3_bucket\":\"$S3_BUCKET_NAME\",\"secret_access_key\":\"$S3_AWS_SECRET_ACCESS_KEY\"}}"
```

Comment:
This will need to use the command [here](https://github.com/graxinc/terraform-aws-grax/blob/66769c58c178b16c02e234d02397f66db8478155/modules/asg/cloud-init.yml#L133) otherwise it will overwrite the storage config on every boot.

Reply:
Going to close this PR but FYI we want that b.c the add-on provider can update the vars, which triggers a reboot - have tested that via triggering a config var update via bucketeer admin

Reply:
@rlewis-grax I think `graxctl config-storage` is missing the ability to inject the access key and secret. The AWS stuff didn't need that since its an instance role.

Reply:
> missing the ability to inject the access key and secret

That appears to be the case, but it could easily be added.